test/resnet/resnet_torchvision.py

```diff
@@ -0,0 +1,43 @@
+"""Test whether torchvision is extendable with graph utils."""
+from test.utils.skip_test import skip_pytorch_below_1_9_0
+from typing import Tuple
+
+import pytest
+import torchvision.models
+from torch import Tensor, rand, rand_like
+from torch.nn import Module, MSELoss
+
+from backpack import backpack, extend
+from backpack.extensions import DiagGGNExact
+
+
+@pytest.fixture
+def model_and_input() -> Tuple[Module, Tensor]:
+    """Yield ResNet model and an input to it.
+
+    Yields:
+        model and input
+    """
+    skip_pytorch_below_1_9_0()
+    resnet18 = torchvision.models.resnet18(num_classes=4).eval()
+    yield extend(resnet18, use_converter=True), rand(2, 3, 7, 7, requires_grad=True)
```

Comment:
Set random seed to make the test deterministic

Reply:
Yes